website_sale_stock_control/tests/test_website_sale_stock_control.py

```diff
@@ -0,0 +1,95 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Sergio Teruel <sergio.teruel@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import urllib
+from lxml import html
+
+from openerp.tests.common import HttpCase
+
+
+class WebsiteSaleStockControlCase(HttpCase):
```

Comment:
Is this working?

Reply:
Yes of course!!! jajaj in new commit :wink: